### PR TITLE
Sections 4 - 9

### PR DESCRIPTION
**Pull request by JuanMaGiraldo**
This PR contains the sections: the router, plug, controller and view & template

### CODE REVIEW

File: lib/hello_web/plugs/locale.ex [hunk 1]
```diff
@@ -0,0 +1,15 @@
+defmodule HelloWeb.Plugs.Locale do
```
Comment (juanhurtado10) [inline lib/hello_web/plugs/locale.ex:1]: @JuanMaGiraldo could you please explain how the Plugs work?

File: lib/hello_web/views/hello_view.ex [hunk 2]
```diff
@@ -0,0 +1,3 @@
+defmodule HelloWeb.HelloView do
```
Comment (juanhurtado10) [inline lib/hello_web/views/hello_view.ex:1]: @JuanMaGiraldo what a `View`does?

**Review by juanhurtado10 — APPROVED** [review]
@JuanMaGiraldo Good job done 👏 . Some questions were added

File: lib/hello_web/views/hello_view.ex (hunk 2 above)
Comment (JuanMaGiraldo) [reply lib/hello_web/views/hello_view.ex:1]: It defines the view layer of the application. View has two functions:
- Render templates.
- View logic: functions that can transform and return data like decorators in RoR.

**Pull request merged**